windows_containers/windows-containers-release-notes-6-x.adoc

```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-6-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 6.0.1
+
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 6.0.0 were released in link:https: //access.redhat.com/errata/RHSA-2023:4488[RHSA-2023:4488].
+
+[NOTE]
+====
+Windows Server 2019 is not supported in vSphere. 
+====
+
+[id="wmco-6-0-1-bug-fixes"]
+=== Bug fixes
+
+* Before this update, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with Windows Defender, regardless of state. This resulted in an error when creating firewall exclusions for 'containerd' on instances without Windows Defender installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed or not. (https://issues.redhat.com/browse/OCPBUGS-3572[**OCPBUGS-3572**])
+
+* Before this update, an endpoint object missing required information caused the WMCO pod to fail during startup. With this fix, WMCO verifies the endpoint object is present with the required fields. As a result, WMCO is able to start and reconcile an invalid or misconfigured endpoint object. (https://issues.redhat.com/browse/OCPBUGS-4336[**OCPBUGS-4336**]) 
+
+* Before this startup, the `containerd` container runtime reported an incorrect version on each Windows node because repository tags were not propagated to the build system. This configuration caused `containerd` to report its Go build version as the version for each Windows node. With this update, the correct version is injected into the binary during build time, so that `containerd` reports the correct version for each Windows node. (link:https://issues.redhat.com/browse/OCPBUGS-8055[*OCPBUGS-8055*])
```

Comment:
```suggestion
* Before this update, the `containerd` container runtime reported an incorrect version on each Windows node because repository tags were not propagated to the build system. This configuration caused `containerd` to report its Go build version as the version for each Windows node. With this update, the correct version is injected into the binary during build time, so that `containerd` reports the correct version for each Windows node. (link:https://issues.redhat.com/browse/OCPBUGS-8055[*OCPBUGS-8055*])
```

Reply:
I think it should be "Before this update" instead of "Before this startup".